programs/near_call.zasm

```diff
@@ -0,0 +1,25 @@
+	.text
+	.file	"near_call.zasm"
+	.globl	__entry
+
+__entry:
+.func_begin0:
+	; gas near call = 25
+	near_call r0, @__call , @__end
+	; gas sstore = 5511
+	sstore	r0, r1
+	; gas ret = 5
+	ret
+.func_end0:
+__end:
+__call:
+.func_begin1:
+	; gas add = 6
+	add 5,r0,r1
+	; gas ret = 5
+	ret
+.func_end1:
+
+
+	.note.GNU-stack
+	.rodata
```

Comment:
Is gas consumption being tracked properly?
```rs
#[test]
fn test_near_call() {
    let bin_path = make_bin_path_asm("near_call");
    let (result, vm) = run_program(&bin_path);
    let expected_gas_left = DEFAULT_INITIAL_GAS - (25 + 5111 + 5 + 6 + 5);
    assert_eq!(expected_gas_left, vm.gas_left());
    assert_eq!(result, U256::from(5));
}
```
Fails:
```bash
assertion `left == right` failed
  left: 60384
 right: 59984
 ```

Reply:
The `sstore` opcode has a gas cost of 5511, you put 5111, the output value is actually correct, I'll add this test so we make sure it doesn't break

Reply:
You are right, did not notice the typo, sorry for the mistake!